### ISSUE TITLE
View with a $where can't be aliased in .drift files

### ISSUE THREAD

**Issue by North101** [simolus3/moor#2153]
Views can't be aliased in .drift files. I get the following error:

```
lib/db/database.g.dart:2677:32: Error: The argument type 'MyTableView' can't be assigned to the parameter type 'TableInfo<Never, dynamic>'.
 - 'MyTableView' is from 'package:botc_helper/db/database.dart' ('lib/db/database.dart').
 - 'TableInfo' is from 'package:drift/src/runtime/query_builder/query_builder.dart' ('../../.pub-cache/git/drift-75fe463a9fa5d344cff623828aec4b6bd5409205/drift/lib/src/runtime/query_builder/query_builder.dart').
        where?.call(alias(this.myTableView, 'my_table')) ??
                               ^
```

Minimal example:
```sql
CREATE TABLE my_table(
  id TEXT NOT NULL PRIMARY KEY,
  name TEXT NOT NULL
);

CREATE VIEW my_table_view AS
SELECT *
FROM my_table;

list($where = TRUE):
SELECT *
FROM my_table_view AS my_table
WHERE $where;
```

Without the WHERE $where it works as expected

**Comment by North101** [comment]
ok, I should have tested the minimal example 1st. Give me a few min

**Comment by North101** [comment]
Fixed example

**Comment by simolus3** [comment]
Thanks a lot for the report! This is fixed in b7690e84d9e1a0b7825c740eafbd375c9875774d.
